libp2p/NodeTable.cpp

```diff
@@ -514,16 +508,17 @@ void NodeTable::onReceived(UDPSocketFace*, bi::udp::endpoint const& _from, bytes
 			{
 				PingNode in = PingNode::fromBytesConstRef(_from, rlpBytes);
 				if (in.version != dev::p2p::c_protocolVersion)
+					return;
+				if (RLPXDatagramFace::secondsSinceEpoch() - in.ts > 3)
```

Comment:
A packet with an expiration > current time is valid, all other packets are invalid. At least that is how 
all other implementations work.

The timestamps are `uint32` and the one sent by the other end is usually bigger than the current time.
Subtracting the remote timestamp from the current time will overflow into a very big number, causing
the packet to be considered invalid.